modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/encoder/BooPickleInstances.scala

```diff
@@ -27,7 +28,7 @@ trait BooPickleInstances {
     }
 
   def booEncoderOf[A](implicit encoder: Pickler[A]): EntityEncoder[A] =
-    EntityEncoder[ByteBuffer].contramap[A] { v =>
-      Pickle.intoBytes(v)
+    EntityEncoder[ByteVector].contramap[A] { v =>
```

Comment:
`ByteBuffer` comes from java nio and its behaviour is complicated as it is mutable. `ByteVector` comes from scodec and it is immutable so `http4s` will prefer to use it instead. `EntityEncoder[ByteBuffer]` will be removed altogether on `0.17.x`